tensorboard/plugins/profile/overview_page/overview-page.html

```diff
@@ -176,37 +176,66 @@
          value: false,
          notify: true,
        },
+       _show_step_time_graph: {
+         type: Boolean,
+         value: false,
+         notify: true,
+       },
        _top_ops_button_text: {
          type: String,
          computed: '_getTopOpsButtonText(_show_top_ops_table)'
        },
+       _generalAnalysisJson: {
+         type: String,
+         notify: true,
+       },
+       _inputAnalysisJson: {
+         type: String,
+         notify: true,
+       },
+       _runEnvironmentJson: {
+         type: String,
+         notify: true,
+       },
+       _recommendationJson: {
+         type: String,
+         notify: true,
+       },
        _show_error_message: {
          type: Boolean,
-         computed: '_shouldShowErrorMessage(_data)',
+         value: false,
          notify: true,
-         observer: '_onErrorChanged',
        },
        _active: {
          type: Boolean,
          value: false,
          notify: true,
+         observer: '_onActiveChanged',
        },
-       _host_idle_time_percent: { type: String, },
-       _device_idle_time_percent: { type: String, },
-       _mxu_utilization_percent: { type: String, },
-       _steptime_ms_average: { type: String, },
-       _steptime_ms_stddev: { type: String, },
-       _top_ops_heading: { type: String, },
-       _error_message: { type: String, },
-       _host_count: { type: String, },
-       _tpu_type: { type: String, },
-       _tpu_core_count: { type: String, },
-       _batch_size: { type: String, },
-       _change_list: { type: String, },
-       _build_time: { type: String, },
-       _build_target: { type: String, },
-       _statement: { type: String, },
+       _host_idle_time_percent: { type: String, notify: true, },
```

Comment:
underscore in the property name kinda indicates private method yet this is notifying for parent/ancestor consumption. Is this correct?

Reply:
Done, removed the notifies. Those are private members.